lib/trento_web/controllers/v1/suma_credentials_controller.ex

```diff
@@ -116,7 +112,4 @@ defmodule TrentoWeb.V1.SUMACredentialsController do
   end
 
   def get_policy_resource(_), do: Trento.SoftwareUpdates.Settings
-
-  defp decode_body(body) when is_struct(body), do: Map.from_struct(body)
```

Comment:
Is this the unique real place where we are forced to decode to map?
As far as I see, we can get the remaining fields exactly the same way without the `struct?` field